pages.zh/common/javadoc.md

```diff
@@ -0,0 +1,16 @@
+# javadoc
+
+> 从源代码以 HTML 格式生成 Java API 文档。
+> 更多信息见：https://docs.oracle.com/javase/9/javadoc/javadoc-command.htm>.
+
+- 生成 Java 源代码的文档并将结果保存在文件夹中：
+
+`javadoc -d {{path/to/directory/}} {{path/to/java_source_code}}`
+
+- 生成指定编码的文档：
+
+`javadoc -docencoding {{UTF-8}} {{path/to/java_source_code}}`
+
+- 生成文档（不包括某些软件包）：
```

Comment:
I don't think the `excluding some packages` should go in parentheses here.

Reply:
Okay, I have fixed it and make it clearer @marchersimon